CONTRIBUTING.md

```diff
@@ -37,38 +37,33 @@ The following instructions uses `$VERSION` as a placeholder, where `$VERSION` is
     }
     ```
 
-1. Set the version in `lib/dnsimple.js`:
+3. Set the version in `lib/dnsimple.js`:
 
     ```javascript
     Dnsimple.VERSION = '$VERSION';
     ```
 
-1. Run the test suite and ensure all the tests pass.
+4. Run the test suite and ensure all the tests pass.
 
-1. Finalize the `## main` section in `CHANGELOG.md` assigning the version.
+5. Finalize the `## main` section in `CHANGELOG.md` assigning the version.
 
-1. Commit and push the changes
+6. Commit and push the changes
 
     ```shell
     git commit -a -m "Release $VERSION"
     git push origin main
     ```
 
-1. Wait for CI to complete.
+7. Wait for CI to complete.
```

Comment:
It's a minor thing, but keeping the number at `1` for all steps makes it easier to maintain the list sequence since you don't have to worry about what's the value of each number. Markdown does this for you.

Reply:
That's a good point (and I didn't know about that)! Thank you!

Reply:
I agree. Please remember to roll it back before merging @ecomba